tests/encoder_decoder/test_modeling_flax_encoder_decoder.py

```diff
@@ -305,7 +306,7 @@ def check_pt_flax_equivalence(self, pt_model, fx_model, inputs_dict):
         fx_outputs = fx_model(**inputs_dict).to_tuple()
         self.assertEqual(len(fx_outputs), len(pt_outputs), "Output lengths differ between Flax and PyTorch")
         for fx_output, pt_output in zip(fx_outputs, pt_outputs):
-            self.assert_almost_equals(fx_output, pt_output.numpy(), 1e-5)
+            self.assert_almost_equals(fx_output, pt_output.numpy(), 4e-2)
```

Comment:
@ydshieh is `1e-5` now the default testing precision?

Reply:
Yes. So far for anything higher than `1e-5`, I was able to find some issues, either in the models, or in the model testes.